src/test/java/core/basesyntax/service/ParserServiceImplTest.java

```diff
@@ -0,0 +1,33 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.ParserServiceImpl;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class ParserServiceImplTest {
+    private static final String INPUT_FILE_PATH = "src/test/resources/input.csv";
+    private ReaderServiceImpl readerService;
```

Comment:
```suggestion
```
Manually prepare List<String> to test it. ParserServiceImplTest will fall if there is a mistake in ReaderServiceImpl. It is not correct for Unit test idea.